tests/util/test_comprehensive_json.py

```diff
@@ -0,0 +1,39 @@
+__license__ = "GNU Affero General Public License http://www.gnu.org/licenses/agpl.html"
+__copyright__ = "Copyright (C) 2018 The OctoPrint Project - Released under terms of the AGPLv3 License"
+
+import time
+
+from frozendict import frozendict
+
+from octoprint.util import comprehensive_json
+
+
+def test_bytes_roundtrip():
+    val = b"\x00\x01\x02\x03"
+    res = comprehensive_json.dumps(val)
+    assert res == '"\\u0000\\u0001\\u0002\\u0003"'
+    assert comprehensive_json.loads(res) == "\x00\x01\x02\x03"
```

Comment:
I'm not super happy about `bytes` not round-tripping properly, but I've refrained from changing this in case there are compatibility concerns elsewhere.

Reply:
It now roundtrips properly on the serializing implementation 🥳